telluric/rasterization.py

```diff
@@ -34,9 +34,14 @@ def rasterize(shapes, crs, bounds=None, dest_resolution=None, *, fill_value=None
     if nodata_value is not None:
         warnings.warn(NODATA_DEPRECATION_WARNING, DeprecationWarning)
 
-    if not(dest_resolution):
+    if not dest_resolution:
         raise ValueError("dest_resolution must be specified")
 
+    elif isinstance(dest_resolution, (int, float)):
+        rx = ry = dest_resolution
```

Comment:
I don't understand this case please explain

Reply:
The case `rx = ry = dest_resolution` is the case we were supporting before. The new case is when `rx != ry` (next block).

Reply:
why when dest_resolution is instance of (int, float) it means they are equal?

Reply:
When it's a single number (either an `int` or a `float`) we have to assume it's the same for X and Y. This was what we were doing so far.